File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-5.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length == 0) return 0;

        int placer = 0;

        for (int comparer = 1; comparer < nums.length; comparer++) {
            if (nums[comparer] != nums[placer]) {
                placer++;
                nums[placer] = nums[comparer];
            }
        }

        return placer + 1;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-5.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if nums == []:
            return 0
        seen = set()

        for num in nums:
            seen.add(num)

        longest = 1
        for num in seen:
            currseq = 1
            nextnum = num+1
            while nextnum in seen:
                currseq += 1
                nextnum += 1
            longest = max(longest, currseq)
        
        return longest
            